docs/_data/relatedItems.yaml

```diff
@@ -49,6 +49,10 @@ rh-health-index:
  - rh-avatar
  - rh-badge
  - rh-tag
+rh-icon: 
+ - rh-alert
+ - rh-card
+ - rh-navigation-secondary
```

Comment:
Change `rh-navigation-secondary` to `rh-statistic`. I had forgotten to update this section in the mockup. 🤦🏽‍♀️